src/i_rumble.c

```diff
@@ -767,6 +770,6 @@ void I_DisableRumble(void)
 
 void I_BindRumbleVariables(void)
 {
-    BIND_NUM_GENERAL(joy_rumble, 0, 0, MAX_RUMBLE_SCALE,
-        "Rumble intensity (0 = Off; 1 = Low; 2 = Medium; 3 = High)");
+    BIND_NUM_GENERAL(joy_rumble, 5, 0, 10,
+        "Rumble intensity (0 = Off; 10 = 100%)");
```

Comment:
@rfomin I think the last few commits should give good results.

Reply:
Yes, this is much better, thank you!

A few nitpicks:
 * Is it just my controller, or does the SSG feels weaker than the regular SG?
 * The pistol is probably too powerful. Maybe we should adjust the rumble based on damage, not just sound?

Reply:
I tested some more and it feels great! The mod support seems to be magical 😮

Reply:
> A few nitpicks:

I made some adjustments here: https://github.com/fabiangreffrath/woof/pull/1876/commits/2f4402b1fdb89ea8c59df25758d9ac7effb1ffa6

That's as much as I can do without making other gamepads feel unbalanced.

Reply:
Thank you very much, the pistol feels balanced now. I hope you didn't compromise on this by making things worse for other gamepads, because I guess not many XBox360 gamepads are still alive.

I've got a lot of fun playing it with various mods 😄 WADs with interesting weapon sets:
1. [The Quinary Conundrum](https://www.doomworld.com/idgames/levels/doom2/Ports/megawads/tqc)
2. [Deadliest Dem(o(li)ti)on](https://www.doomworld.com/idgames/levels/doom2/d-f/demolit)
3. [Crusader](https://www.doomworld.com/forum/topic/139104-crusader-dsdhackedmbf21/)

Reply:
Thanks for the example wads, it's nice to have more for testing.

Reply:
To me,  [Deadliest Dem(o(li)ti)on](https://www.doomworld.com/idgames/levels/doom2/d-f/demolit) is one of the most creative and fun WADs ever, it's criminally overlooked. Just play the first level legitimately for a while, I guarantee you'll smile. And you might even want to tweak the rumble some more 😄

Reply:
> And you might even want to tweak the rumble some more 😄

I already noticed the chainsaw rumble is very intense in Crusader. One limiting factor is MBF21's `A_WeaponSound`. It provides no information about what is calling it. Is it the weapon firing? Is reloading sounds? I think there's no way to tell. I'll try to tune things some more later after reviewing those example wads.

Reply:
> I'll try to tune things some more later after reviewing those example wads.

Just in case, to me rumble is already very good 👍 

> One limiting factor is MBF21's `A_WeaponSound`. It provides no information about what is calling it. Is it the weapon firing? Is reloading sounds? I think there's no way to tell.

Yes, I think there's no way to find out without changing the standard.

Reply:
> Just in case, to me rumble is already very good 👍

I experimented but I'll leave it alone for now. I think this is the best compromise given the limitations of the standard.